src/main/java/com/github/devotedmc/hiddenore/listeners/BlockBreakListener.java

```diff
@@ -386,27 +374,27 @@ private void doActualGenerate(final List<ItemStack> items, final Location source
 						
 				log("STAT: Player {0} at {1} broke {2} - replacing with {3} {4} as {6}", 
 						player.getDisplayName(), player.getLocation(), blockName, 
-						xform.getAmount()- cPlace, name, expressed);
+						placed, name, expressed);
 				
 				// Anything to tell anyone about?
-				if (cPlace < xform.getAmount() && Config.isAlertUser()) {
-					if (blockConfig.hasCustomPrefix(dropName)) {
-						StringBuffer customAlerts = new StringBuffer(blockConfig.getPrefix(dropName));
-						
-						customAlerts.append(" ").append(xform.getAmount() - cPlace).append(" ").append(
-								name).append(" nearby"); // TODO: Replace with configured suffix
-						player.sendMessage(ChatColor.GOLD + customAlerts.toString());
-					} else {
-						if (Config.isListDrops()) {
-							alertBuffer.append(" ").append(xform.getAmount() - cPlace).append(" ").append(
-									name).append(" nearby,"); // TODO: Replace with configured suffix
-						}
-					}
+				if (placed > 0 && Config.isAlertUser()) {
+					alertUser(player, blockConfig, dropName, placed, name);
 				}
 			}
 		}
 	}
 	
+	private boolean alertUser(Player player, BlockConfig blockConfig, String dropName, int amount, String itemName){
+		if (blockConfig.hasCustomPrefix(dropName)) {
+			StringBuffer customAlert = new StringBuffer(blockConfig.getPrefix(dropName));
```

Comment:
StringBuffers are a concurrent alternative to StringBuilder. If you don't expect concurrent access, always use StringBuilder over StringBuffer